src/store/modules/user/getters.ts

```diff
@@ -9,6 +9,11 @@ const getters: GetterTree <UserState, RootState> = {
     isUserAuthenticated(state) {
         return state.token && state.current
     },
+    getBaseUrl(state) {
+        let baseURL = process.env.VUE_APP_BASE_URL;
+        if (!baseURL) baseURL = state.instanceUrl;
+        return baseURL.startsWith('http') ? baseURL : `https://${baseURL}.hotwax.io/api/`;
```

Comment:
Handle case to check for /api in the URL.
https://github.com/hotwax/receiving/blob/main/src/store/modules/user/getters.ts#L15